README.md

```diff
@@ -1,7 +1,11 @@
 # Introduction
 
-[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/OpenMined/PySyft/master) [![Chat on Slack](https://img.shields.io/badge/chat-on%20slack-7A5979.svg)](https://openmined.slack.com/messages/team_pysyft) [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fmatthew-mcateer%2FPySyft.svg?type=small)](https://app.fossa.io/projects/git%2Bgithub.com%2Fmatthew-mcateer%2FPySyft?ref=badge_small)
-![](https://github.com/OpenMined/PySyft/workflows/Run%20tests/badge.svg) ![](https://github.com/OpenMined/PySyft/workflows/Run%20notebook%20tests/badge.svg)
+![](https://github.com/OpenMined/PySyft/workflows/Tests/badge.svg)
+![](https://github.com/OpenMined/PySyft/workflows/Tutorials/badge.svg)
```

Comment:
![](https://github.com/OpenMined/PySyft/workflows/Run%20notebook%20tests/badge.svg)
![](https://github.com/OpenMined/PySyft/workflows/Run%20tests/badge.svg)

Reply:
Fix using this doc: https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository

Reply:
I'm trying to change the text of the badges by updating the workflow names. Only one of the two seems to be working...

Reply:
I think the reason for that is that the `Tutorials` badge doesn't exist yet, because this PR doesn't trigger that workflow. Should be okay to merge anyway, and I can manually trigger the workflow on master to generate the badge.

Reply:
(Alternately, maybe I can manually trigger them on this branch.)

Reply:
Should be good yes :)